bpm/bonita-api/bonita-common-api-impl/src/main/java/org/bonitasoft/engine/bpm/bar/ActorMappingContribution.java

```diff
@@ -35,4 +44,32 @@ public String getFileName() {
         return ACTOR_MAPPING_FILE;
     }
 
+    @Override
+    public boolean readFromBarFolder(final BusinessArchive businessArchive, final File barFolder) throws IOException {
+        final File file = new File(barFolder, ACTOR_MAPPING_FILE);
+        if (file.exists()) {
+            final byte[] content = IOUtil.getContent(file);
+            try {
+                businessArchive.setActorMapping(new ActorMappingConverter().deserializeFromXML(content));
+            } catch (JAXBException | org.xml.sax.SAXException e) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    @Override
+    public void saveToBarFolder(final BusinessArchive businessArchive, final File barFolder) throws IOException {
+        final ActorMapping actorMapping = businessArchive.getActorMapping();
+        if (actorMapping != null) {
+            try {
+                final byte[] fileContent = new ActorMappingConverter().serializeToXML(actorMapping);
+                final File file = new File(barFolder, ACTOR_MAPPING_FILE);
+                IOUtil.write(file, fileContent);
+            } catch (JAXBException | org.xml.sax.SAXException e) {
+                throw new IOException("Cannot write Actor Mapping to Bar folder", e);
+            }
```

Comment:
More than half of the BAR creation tests generate an archive without an ActorMapping. So no error here *a priori* : it was this way before the PR additions; and raising an exeption here will simply mean we will have to ignore it somewhere in the code